src/admin/Admin.tsx

```diff
@@ -1,16 +1,48 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 
 import { Flex } from '@viaa/avo2-components';
 
+import { NavigationItemInfo } from '../shared/types';
+
 import { ADMIN_PATH } from './admin.const';
 import { renderAdminRoutes } from './admin.routes';
 import { Sidebar } from './shared/components';
 
 const Admin = () => {
-	const NAV_ITEMS = [
-		{ label: 'Gebruikers', location: ADMIN_PATH.USER, key: 'users' },
-		{ label: 'Navigatie', location: ADMIN_PATH.MENU, key: 'navigatie' },
-		{ label: 'Content', location: ADMIN_PATH.CONTENT, key: 'content' },
+	const [t] = useTranslation();
+
+	const NAV_ITEMS: NavigationItemInfo[] = [
+		{
+			label: t('Gebruikers'),
+			location: ADMIN_PATH.USER,
+			key: 'users',
+			exact: false,
+			subLink: false,
+		},
+		{
+			label: t('Navigatie'),
+			location: ADMIN_PATH.MENU,
+			key: 'navigatie',
+			exact: false,
+			subLink: false,
+		},
+		{
+			label: t('Content'),
+			location: ADMIN_PATH.CONTENT,
+			key: 'content',
+			exact: false,
+			subLink: false,
```

Comment:
Deze const verhuizen naar `admin.const.ts` en mss ipv subLink een subRoutes/children property gebruiken zodat het duidelijk is waar ze onder hangen?